proposals/Template-Instantiation.md

```diff
@@ -395,45 +392,45 @@ The  `createInstance(optional any state)` method on `HTMLTemplateElement`, when
 2. Let *instance* be an instance of `TemplateInstance`.
 3. [Append](https://dom.spec.whatwg.org/#concept-node-append) *clonedTree* to *instance*.
 4. Let *parts* be an empty list.
-5. For every [descendent](https://dom.spec.whatwg.org/#concept-tree-descendant) node c*urrentNode* of *instance* in [tree order](https://dom.spec.whatwg.org/#concept-tree-order), run these steps:
-    1. If c*urr**entNode* is a [template element](https://html.spec.whatwg.org/multipage/scripting.html#the-template-element):
-        1. Run the concept to _adjust single node case_ with *currentNode***.**
-        2. Let *nodeValueSetter* be a new instance of the _node value setter_ with *currentNode*, the [previous sibling](https://dom.spec.whatwg.org/#concept-tree-next-sibling) of *currentNode*, the [next sibling](https://dom.spec.whatwg.org/#concept-tree-next-sibling) of *currentNode*, an empty _previous replacement nodes_, fully templatized set to the result of running the concept to _determine full templatizability_ with *currentNode*, and an empty _node template part list_.
-        3. Let *innerPart* be a new instance of `InnerTemplatePart` associated with *currentNode*, an empty _replacement node list_, and *nodeValueSetter*.
-        4. Append *innerPart* to the end of *parts.*
-        5. [Remove](https://dom.spec.whatwg.org/#concept-node-remove) *currentNode* from the *currentNode*'s [parent](https://dom.spec.whatwg.org/#concept-tree-parent).
-        6. Run the concept to _apply node template part list_ with *nodeValueSetter*.
+5. For every [descendant](https://dom.spec.whatwg.org/#concept-tree-descendant) node *currentNode* of *instance* in [tree order](https://dom.spec.whatwg.org/#concept-tree-order), run these steps:
+    1. If *currentNode* is a [`template` element](https://html.spec.whatwg.org/multipage/scripting.html#the-template-element):
+      1. Run the concept to _adjust single node case_ with *currentNode*.
+      2. Let *nodeValueSetter* be a new instance of the _node value setter_ with *currentNode*, the [previous sibling](https://dom.spec.whatwg.org/#concept-tree-next-sibling) of *currentNode*, the [next sibling](https://dom.spec.whatwg.org/#concept-tree-next-sibling) of *currentNode*, an empty _previous replacement nodes_, _fully templatized_ flag set to the result of running the concept to _determine full templatizability_ with *currentNode*, and an empty _node template part list_.
```

Comment:
I'm using _ to only refer to variable names. So please don't italicize concepts defined elsewhere.